Server/config.json

```diff
@@ -5,5 +5,8 @@
 				   [0.25, 0.5, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25],
 				   [0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25]],
     "useDropping": true,
-    "dropping": [{"round":1,"#clients":1},{"round":2,"#clients":1} ]
+    "dropping": [{"round":1,"#clients":1},{"round":2,"#clients":1} ],
+
+    "useWeights": true
```

Comment:
@tem556 This is not what we intended. We want the weights to be the same as the data partition ratios. For example, if we divide the dataset 20% : 20% : 60% among the clients then the weights would be 0.2 : 0.2 : 0.6

Reply:
I see, I think I misunderstood